cmd/tink-worker/cmd/container.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"context"
+	"path"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+type containerAction struct {
```

Comment:
what is an action? is this in relation to the Tink concept of an action? If so, we might want to rethink this name? If one of our design goals is for this container functionality to be foundational then we might not want the naming to be coupled to Tink-specific words. Thoughts?